BasicRdl/ViewModels/CategoryBrowserViewModel.cs

```diff
@@ -113,6 +123,25 @@ public bool CanCreateRdlElement
             private set { this.RaiseAndSetIfChanged(ref this.canCreateRdlElement, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the dock layout group target name to attach this panel to on opening
+        /// </summary>
+        public string TargetName { get; set; } = LayoutGroupNames.LeftGroup;
+
+        ///<inheritdoc/>
```

Comment:
We don't use <inheritdoc/>, and while it saves time, we try to maintain docs explicit: i.e. visible at inspection time.